bin/reth/src/node/mod.rs

```diff
@@ -110,11 +123,60 @@ impl Command {
 
         init_genesis(db.clone(), self.chain.clone())?;
 
+        match &self.import {
+            Some(import_path) => {
```

Comment:
This should be a standalone command since we need to import blocks from different files in some tests, please move it as the original issue stipulates

Reply:
refactored to be a standalone command! import currently has a `--path` argument though, would prefer to change to `reth import chain.rlp` in a followup though if that's the desired UX